build.gradle

```diff
@@ -36,9 +36,9 @@ runIdeForUiTests {
 }
```

Comment:
`build.gradle` needs to be updated to depend on the bumped intellij-kubernetes once its PR https://github.com/redhat-developer/intellij-kubernetes/pull/548 is merged
```
plugins = [...'com.redhat.devtools.intellij.kubernetes:0.7.1.XXX']
```